### DISCUSSION TITLE
Wordpress Kadence Theme/Blocks - No Scroll Bars On Viewport

### DISCUSSION THREAD

**Discussion by pcboard** [PrismJS/prism#3076] [Q&A]
Having a bit of a challenge with Kadence Theme/Blocks with PrismJS. Everything seems to work fine (themes/colors), with the exception of the displayed viewport size. The displayed code is always full in size matching the number of rows in the code being displayed.

Samples/examples seem to show about 20 lines of text in the viewport with horizontal and vertical scroll bars when the text is larger than the viewport size. The install we have made always displays a large viewport, with dozens of lines being displayed (large source file was being displayed) without any scroll bars.

I believe this has something to do with Kadence setting the <pre> style, but I am stumped how to overcome this.

Any thoughts or ideas on overcoming this?

Thanks in advance...

**Comment by RunDevelopment** [comment]
I'm sorry but this probably isn't the right place to ask this question. From what you describe, I don't think your problem is caused by Prism or its themes. It's probably a problem with Kadence or its settings as you suggested. 

Unfortunately, we can't help you with this problem.
